main.py

```diff
@@ -12,7 +12,7 @@
 
 from kytos.core import KytosEvent, KytosNApp, log, rest
 from kytos.core.common import EntityStatus
-from kytos.core.exceptions import KytosLinkCreationError
+from kytos.core.exceptions import KytosLinkCreationError, KytosResizingAvailableTagError
```

Comment:
Scrutinizer will fail mostly because this exception is new. When assigning new `tag_ranges`, `available_tags` needs to be resized to comply with new restrictions. When it is not possible (e.g. a used tag is not included in `tag_ranges`) `KytosResizingAvailableTagError` will be raised.

Reply:
You can parametrize temporarily a branch with `@<branch_name` if you have a kytos upstream branch check out this example here https://github.com/kytos-ng/topology/pull/129/commits/574708204a0b999e4260aa681d9f7c3e907aec2f, and then you can revert later on once it gets approved and right before merging it.

Reply:
Added it but I need to refactor tests although now it is easier.